src/validate_pyproject/pre_compile/cli.py

```diff
@@ -58,7 +58,7 @@ def JSON_dict(name: str, value: str) -> Dict[str, Any]:
         flags=("-t", "--tool"),
         action="append",
         dest="tool",
-        help="External tools file/url(s) to load, of the form name=URL#path",
+        help="External tools file/url(s) to load, of the form name=URL#path, name can be empty",
```

Comment:
This looks fine, but I wonder if it would be cleaner to have another separated flag like `--load-schema` or `--referenced-schema` or `--auxiliar-schema`. This way we don't have the oddity of starting a schema with `=`.

Reply:
Fair enough. Do we need to be explicit about this CLI feature then? Or is it OK if it just stay as a "happy little accident". I would avoid committing to a design at this stage without it being requested by the users.